commands/user_e2e_test.go

```diff
@@ -136,3 +136,46 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestUserConvertCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForAllClients("Error when no flag provided", func(c client.Client) {
+		printer.Clean()
+
+		emailArg := "example@example.com"
+		cmd := &cobra.Command{}
+
+		err := userConvertCmdF(c, cmd, []string{emailArg})
+		s.Require().Error(err)
```

Comment:
Shall we check if we have the expected error here?

Reply:
Done.